docs/benchmarks/rabi_oscillations.ipynb

```diff
@@ -0,0 +1,495 @@
+    "\n",
+    "1. Prepare the $|0\\rangle$ state.\n",
+    "2. Rotate by an angle $\\theta$ around the $X$ axis.\n",
+    "3. Measure to see if the result is a 1 or a 0.\n",
+    "4. Repeat steps 1-3 $k$ times.\n",
+    "5. Report the fraction of $\\frac{\\text{Number of 1's}}{k}$\n",
+    "found in step 3."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ACqqV6tJ7xXp"
+   },
+   "source": [
+    "## 1. Getting to know Cirq\n",
+    "\n",
+    "Cirq emphasizes the details of implementing quantum algorithms on near term devices.\n",
+    "For example, when you work on a qubit in Cirq you don't operate on an unspecified qubit that will later be mapped onto a device by a hidden step.\n",
+    "Instead, you are always operating on specific qubits at specific locations that you specify.\n",
+    "\n",
+    "Suppose you are working with a 54 qubit Sycamore chip.\n",
+    "This device is included in Cirq by default.\n",
+    "It is called `cirq_google.Sycamore`, and you can see its layout by printing it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "rKoMKEw46XY7"
+   },
+   "outputs": [],
+   "source": [
+    "import cirq\n",
+    "import cirq_google\n",
+    "\n",
+    "working_device = cirq_google.Sycamore\n",
+    "print(working_device)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FJJEbuk-98Gj"
+   },
+   "source": [
+    "For this experiment you only need one qubit and you can just pick whichever one you like."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "XoXekxuQ8bI0"
+   },
+   "outputs": [],
+   "source": [
+    "my_qubit = cirq.GridQubit(5, 6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8Tucm7os-uET"
+   },
+   "source": [
+    "Once you've chosen your qubit you can build circuits that use it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "niH8sty--Hu0"
+   },
+   "outputs": [],
+   "source": [
+    "from cirq.contrib.svg import SVGCircuit\n",
+    "\n",
+    "# Create a circuit with X, Ry(pi/2) and H.\n",
+    "my_circuit = cirq.Circuit(\n",
+    "    # Rotate the qubit pi/2 radians around the X axis.\n",
+    "    cirq.rx(3.141 / 2).on(my_qubit),\n",
+    "    # Measure the qubit.\n",
+    "    cirq.measure(my_qubit, key=\"out\"),\n",
+    ")\n",
+    "SVGCircuit(my_circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "-zbI-2KUMU66"
+   },
+   "source": [
+    "Now you can simulate sampling from your circuit using `cirq.Simulator`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "IqUn4uv9_IVo"
+   },
+   "outputs": [],
+   "source": [
+    "sim = cirq.Simulator()\n",
+    "samples = sim.sample(my_circuit, repetitions=10)\n",
+    "samples"
```

Comment:
I don't think you need this line since the previous line should print out the samples.